src/appleseed/foundation/image/genericimagefilewriter.h

```diff
@@ -33,29 +32,67 @@
 // appleseed.foundation headers.
 #include "foundation/image/iimagefilewriter.h"
 #include "foundation/image/imageattributes.h"
+#include "foundation/image/pixel.h"
 
-// appleseed.main headers.
-#include "main/dllsymbol.h"
+// OpenImageIO headers.
+#include "foundation/platform/_beginoiioheaders.h"
+#include "OpenImageIO/imageio.h"
```

Comment:
If all you need is the declaration of `OIIO::ImageOutput`, maybe you can introduce a forward declaration? That would prevent pulling in OIIO in many files thus speeding up compilation.

Reply:
Nop.
OIIO is templated (including ```OIIO::ImageOutput```). I must to include it in the header file.

Reply:
Ok, that's what I feared. Thanks.